src/routes/paths.ts

```diff
@@ -13,3 +13,37 @@ export const APPLICATION_LIST_PATH = WORKSPACE_PATH.extend(`applications`);
 export const APPLICATION_DETAILS_PATH = APPLICATION_LIST_PATH.extend(
   `:${RouterParams.applicationName}`,
 );
+
+export const APPLICATION_RELEASE_LIST_PATH = APPLICATION_DETAILS_PATH.extend(`releases`);
+
+export const APPLICATION_RELEASE_DETAILS_PATH = APPLICATION_RELEASE_LIST_PATH.extend(
+  `:${RouterParams.releaseName}`,
+);
+
+export const APPLICATION_SNAPSHOT_LIST_PATH = APPLICATION_DETAILS_PATH.extend(`snapshots`);
+
+export const APPLICATION_SNAPSHOT_DETAILS_PATH = APPLICATION_SNAPSHOT_LIST_PATH.extend(
+  `:${RouterParams.snapshotName}`,
+);
+
+export const APPLICATION_PIPELINERUN_LIST_PATH = APPLICATION_DETAILS_PATH.extend(`pipelineruns`);
+
+export const APPLICATION_PIPELINERUN_DETAILS_PATH = APPLICATION_PIPELINERUN_LIST_PATH.extend(
+  `:${RouterParams.pipelineRunName}`,
+);
+
+export const RELEASE_LIST_PATH = WORKSPACE_PATH.extend(`release`);
+
+export const RELEASE_DETAILS_PATH = RELEASE_LIST_PATH.extend(`:${RouterParams.releaseName}`);
```

Comment:
Where are these routes being used?

Reply:
The first one is used by:
![Screenshot 2025-02-24 at 17 24 32](https://github.com/user-attachments/assets/5f1b94f6-3814-4093-9543-f7c8c273e46e)
The second one is useless. I would delete it.